src/util/expr_util.cpp

```diff
@@ -18,19 +18,20 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "pointer_expr.h"
 #include "std_expr.h"
 
-bool is_lvalue(const exprt &expr)
+bool is_assignable(const exprt &expr)
```

Comment:
Should this perhaps be moved to goto-programs? The use in `replace_symbol` obviously conflicts with that, but some follow-up cleanup may be necessary.